### PR TITLE
Week 13: React native app

### PR DESCRIPTION
**Pull request by monkeybusiness101**
My reflections on how this week's project turned out:

A bit short with one week 

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,24 +1,80 @@
...
+const Text = styled.Text`
+  align-items: center;
+  justify-content: center;
+  color: white;
+`
+const FetchButton = styled.TouchableOpacity`
+  background-color: tomato;
+  align-items: center;
+  justify-content: center;
+  fontWeight: bold;
```
Comment (EmOlsz) [inline App.js:26]: We shouldn't use camelCase in styled component, `font-weight` is proper way

File: App.js [hunk 2]
```diff
@@ -1,24 +1,80 @@
...
+  color: white;
+  width: 100px;
+  height: 50px;
+`
+const TextField = styled.TextInput`
+height: 40;
+width: 200;
```
Comment (EmOlsz) [inline App.js:33]: Wrong indentations

File: App.js [hunk 3]
```diff
@@ -1,24 +1,80 @@
...
+margin: 5px;
+borderColor: tomato;
+borderWidth: 3;
+textAlign:center;
 `
 
 const App = () => {
+
+  const [fetched, setFetched] = useState(false)
+  const [value, onChangeText] = useState('Fireball');
```
Comment (EmOlsz) [inline App.js:43]: This pair should rather be named `value` and `setValue`

File: App.js [hunk 4]
```diff
@@ -1,24 +1,80 @@
...
+      "method": "GET",
+      "headers": {
+        "x-rapidapi-key": "9e4bc04d24mshe167fe2dbc59904p1dda1ejsnb8edfa52d102",
+        "x-rapidapi-host": "omgvamp-hearthstone-v1.p.rapidapi.com"
+      }
+    })
+    .then((response) => response.json())
+    .then(response => {
+      getData(response)
+      console.log(response)
```
Comment (EmOlsz) [inline App.js:58]: We should remove console.logs before final push

File: App.js [hunk 5]
```diff
@@ -1,24 +1,80 @@
...
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
+      { fetched === true ? (
+      data.map( (card) => {
+        return <Card source={{uri:`${card.img}`,}} key={card.cardId} />
+      })      
+      )  : null}
+      <TextField
+      onChangeText={text => onChangeText(text)}
+      value={value}
```
Comment (EmOlsz) [inline App.js:75]: Wrong indentations over here as well

**Review by EmOlsz — CHANGES_REQUESTED** [review]

File: App.js (hunk 1 above)
Comment (monkeybusiness101) [reply App.js]: changed all camelCase to kebab-case

File: App.js (hunk 2 above)
Comment (monkeybusiness101) [reply App.js]: fixed

File: App.js (hunk 3 above)
Comment (monkeybusiness101) [reply App.js]: fixed

File: App.js (hunk 4 above)
Comment (monkeybusiness101) [reply App.js]: fixed

File: App.js (hunk 5 above)
Comment (monkeybusiness101) [reply App.js]: fixed

**Comment by monkeybusiness101** [comment]
@EmOlsz Hi! Thank you for the feedback, I have pushed fixes, can you please review and approve?

**Comment by EmOlsz** [comment]
Hey Max, thanks for changes, reviewed and approved!

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**